internal/report.go

```diff
@@ -192,6 +209,9 @@ func buildTableReportBody(conv *Conv, srcTable string, issues map[string][]Schem
 				}
 				srcType := srcSchema.ColDefs[srcCol].Type.Print()
 				spType := spSchema.ColDefs[spCol].T.PrintColumnDefType()
+				srcName := fmt.Sprintf(srcSchema.ColDefs[srcCol].Name)
```

Comment:
sprintf is used for [formatting strings](https://www.geeksforgeeks.org/fmt-sprintf-function-in-golang-with-examples/). For assigning a string variable like this, we dont need an Sprintf.

Reply:
got it.